src/platform-includes/session-replay/install/javascript.mdx

```diff
@@ -1,13 +1,13 @@
-Install the Replay package with the package manager of your choice. Alternatively, you can load the Replay integration using a CDN bundle:
+The Replay Integration is **already included** in your Browser or Framework SDK NPM packages. If you're using CDN bundles instead of NPM packages, you need to load the Replay integration CDN bundle in addition to your browser bundle:
```

Comment:
```suggestion
The Replay integration is **already included** in your browser or framework SDK NPM packages. If you're using CDN bundles instead of NPM packages, you need to load the Replay integration CDN bundle in addition to your browser bundle:
```